regimes/pt/tax_categories.go

```diff
@@ -135,6 +144,7 @@ var taxCategories = []*tax.Category{
 					i18n.PT: "Artigo 16.°, n.° 6 do CIVA",
 				},
 				Codes: cbc.CodeSet{
+					KeyATTaxCode:          "ISE",
```

Comment:
I'm surprised the linter didn't pick up on this... might be better to have constants when the code is repeated so much?

Reply:
Repeated codes extracted to constants in 82b32dbb5066a08dc8417aaf22532d6edbe4608e. Thank you for the feedback!